src/table/utils/handle-accessibility.ts

```diff
@@ -94,7 +97,7 @@ export const handleResetFocus = ({
       const hasSelectedClassName = cell?.classList?.contains('selected');
       announce({
         keys: [
-          `${cell.textContent},`,
+          `${cell?.textContent},`,
```

Comment:
not sure if it is correct... the added test only works if I set this optional

Reply:
good question. In getCellElement we cast the result to HTMLCellElement, but it seems like it might be undefined. I'll look into it

Reply:
We always assume that `getCellElement` gets a valid `cellCoord`, so it will return a cell. What you need to do in your tests is to mock `rootElement` as in the test above it

Reply:
Yes, I had tried it before and didn't work, also added a mocked cell textContent and didn't pass either. the problem seems is  `totalsPosition`, without that it works fine. I've no idea what change it makes that ends up to an undefined cell?

Reply:
hmm ok let me have a look

Reply:
the reason is that there is now third row in the mock (index 2).
``` 
      rootElement = {
        getElementsByClassName: () => [row, row],
        querySelector: () => cell,
      } as unknown as HTMLElement;
```
As you see here we add only two rows in the return of getElementsByClassName, so it will be undefined for index 2. Add an extra row there and it works